catalog/ui/catalog-ui-search/src/main/webapp/js/widgets/cesium.mapclustering.js

```diff
@@ -174,11 +170,21 @@ define(['backbone',
             var distanceInPixels = distancePerPixel * entity.radius;
             return distanceInPixels;
         },
-        getEllipsoidAt40PercentOfMapView: function(width, height) {
+        getEllipsoidAt40PercentOfMapView : function(width, height) {
             return viewer.camera.pickEllipsoid(new Cesium.Cartesian2(width * 40 / 100, height / 2));
         },
-        getEllipsoidAt60PercentOfMapView: function(width, height) {
+        getEllipsoidAt60PercentOfMapView : function(width, height) {
             return viewer.camera.pickEllipsoid(new Cesium.Cartesian2(width * 60 / 100, height / 2));
+        },
+        getSvgImage : function(value) {
+            var svgString = '<svg xmlns="http://www.w3.org/2000/svg" width="25" height="25">';
```

Comment:
We might want to use a handlebars template here. Something like: 

```javascript
var svg = hbs.compile(template)({
  fill: value.color.toCssColorString(),
  count: value.points.length
});

return window.btoa('data:image/svg+xml;base64,' + svg);
```

would work.